README.md

```diff
@@ -1,23 +1,108 @@
 <!---
-  Copyright 2023 SECO Mind Srl
+  Copyright 2023-2024 SECO Mind Srl
   SPDX-License-Identifier: Apache-2.0
 -->
 
-# EdgehogDeviceForwarder
+# Edgehog Device Forwarder
 
-To start your Phoenix server:
+Proxy server to facilitate communication with Edgehog devices.
+It handles HTTP messages and WebSocket connections between the device and the user client.
 
-  * Run `mix setup` to install and setup dependencies
-  * Start Phoenix endpoint with `mix phx.server` or inside IEx with `iex -S mix phx.server`
+It's only been tested with simple WebSocket connections such as
+[ttyd](https://github.com/tsl0922/ttyd), however it should handle arbitrary requests
+as long as you keep in mind the [limitations](#known-issues-and-limitations).
```

Comment:
nit: opt for the most specific determiner if it's known to the reader.

as long as you keep in mind ~~the~~ _its_ limitations